src/components/Modal/__stories__/helpers.js

```diff
@@ -0,0 +1,39 @@
+import { Button, Flex } from "components";
+import React, { useRef, useState } from "react";
+
+// internal custom hook to help with writing tests and stories.
+export const useModalHelper = ({ openButtonTitle = "Open", openOnStart = false, ...props } = {}) => {
+  const [show, setShow] = useState(openOnStart);
+  const buttonRef = useRef(null);
+  const closeModal = () => setShow(false);
+
+  const openModalButton = (
+    <Button onClick={() => setShow(true)} ref={buttonRef}>
+      {openButtonTitle}
+    </Button>
+  );
+
+  const modalButtons = <ModalButtons closeModal={closeModal} />;
+
+  const modalProps = {
+    ...props,
+    show,
+    onClose: closeModal,
+    triggerElement: buttonRef.current
+  };
+
+  return { modalProps, openModalButton, modalButtons, show, closeModal, ref: buttonRef };
+};
+
+export const ModalButtons = ({ closeModal }) => {
+  return (
```

Comment:
I guess we usualy will prefer to locate the buttons of every modal in monday in the same location above the modal dialog and with the same gap (for consistency)
Why not create ModalButtonsContainer component for this? (will receive buttons (from vibe) as childrens